Sources/Action/Action+Extensions.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Action+Extensions.swift
+//  Action
+//
+//  Created by Obi Bob on 01.03.19.
+//  Copyright © 2019 CezaryKopacz. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+extension Action {
```

Comment:
I think the extensions has to be public or consumers won't be able to use it

```suggestion
public extension Action {
```